templates/csharp/README.md.twig

```diff
@@ -0,0 +1,43 @@
+# {{ spec.title }} {{sdk.name}} SDK
```

Comment:
We'll update the readme together when I finish to register the library in a package manager